client/src/components/Navbar/Navbar.jsx

```diff
@@ -14,7 +14,7 @@ export default function Navbar() {
       <button className="Logo" type="button" onClick={redirectionToHome}>
         <img
           className="logoImage"
-          src="src/assets/images/Logo_foodtastics.png"
+          src="../../../public/images/Logo_foodtastics.png"
```

Comment:
the correct way to do this is simply : /images/Logo_foodtastics.png

Make sure you have a public folder on your react client folder

Reply:
It´s done